### DISCUSSION TITLE
How is balance difference calculated?

### DISCUSSION THREAD

**Discussion by pwsoftwaredesigns** [firefly-iii/firefly-iii#5677] [Firefly III]
I have an account which I have completely reconciled with no "reconciliation" transactions needed.  Thus, my balance between my actual bank and what is in firefly are the same.  However, on the asset accounts page, the "balance difference" column contains a value other than zero. The account has a balance of $15,481.02 and the balance difference says -$9,585.21.

How is this value calculated?  Why does it seem to be incorrect?

**Comment by JC5** [comment]
It's the difference between the start of the month and the end of the month.

**Comment by pwsoftwaredesigns** [comment]
So, if I understand correctly, the difference gets “reset” at the start of each month.  So, if my account has $1000 at the start of the month and I withdraw $800 and deposit $200, the total balance difference for that month would be 1000 - (1000 - 800 + 200) = 400?

**Reply by JC5** [reply]
Yes
